paper.md

```diff
@@ -97,20 +101,24 @@ All course materials were designed to be modular. For example, if students are k
 
-Links to example solutions for all exercises are provided on each page.
+Students have the option to interact with each tutorial notebook using one of two ways. The first option entails [installing Anaconda 3](https://docs.anaconda.com/anaconda/install/) on their local computers with a [custom environment](https://raw.githubusercontent.com/shawnrhoads/gu-psyc-347/master/course-env.yml) made for this course and then opening each notebook using Jupyter Notebook. Instructors can walk students through the installation steps using the instructions provided on the [Getting Started](https://shawnrhoads.github.io/gu-psyc-347/module-00-04_Getting-Started.html) page. However, this can be rather difficult to implement with students' different machines, operating systems, background, etc. As a fail-safe, students also have the option to use [Google Colaboratory](https://colab.research.google.com/), which runs the code on Google's cloud servers for **free**. There is a button at the top of each tutorial page that opens each Jupyter Book page in Google Colaboratory.
+
+Once students are able to run notebooks (locally or on the cloud), they can interact with all of the code in the tutorials. Instructors should guide students through all of the "tutorials", which, for example, will allow them to demonstrate to students how tweaking certain variables in real-time would change the outputs. The "exercises" at the end of each module were created for the students to apply what they have learned. They can work on these by themselves or in groups.
+
+Links to example solutions for all exercises are provided on each page. For additional background on the content covered in each tutorial (technical or theoretical), teachers and students are encouraged to review the references and/or resources listed at the beginning of each tutorial or the non-exhaustive list of resources on the [Resources page](https://github.com/shawnrhoads/gu-psyc-347/blob/master/RESOURCES.md). Content-related questions from students, teachers, and contributors related to the course can also be submitted directly on the GitHub repository. However, we request that individuals make a good faith effort searching for their answer in the relevant readings or resources listed at the beginning of each tutorial before submitting questions. In addition, teachers should familarize themselves with the [recommended readings](https://shawnrhoads.github.io/gu-psyc-347/module-00-03_Reading-List.html) associated with each module and use them to introduce students to each topic, generate discussions and ideas among students, and provide relevant background for each tutorial. For example, teachers should assign the Wilson & Collins (2019) paper for reading and discussion prior to beginning any notebook tutorials for **Module 03**.
```

Comment:
Content after "In addition" could be its own paragraph, since it's addressing teachers specifically

Reply:
> Content after "In addition" could be its own paragraph, since it's addressing teachers specifically

Good catch! I moved around some text here for clarity